ruby/bindings/InitRubyBindings.cpp

```diff
@@ -934,8 +990,11 @@ void locateEmbeddedGems(bool use_bundler) {
       RbConfig::CONFIG['arch'] = original_arch
     end
 
-      Dir.chdir(current_dir)
-    end
+    Dir.chdir(current_dir)
+
+    # clean up the global
+    $ignore_native_gem_names = nil
```

Comment:
At the end I clean up the global, because I'm a good citizen.

In ruby `$<whatever>` returns nil anyways if not defined.

I think we should also do the same for our `$logger` global variable, but not the point there.